stable/nginx-ingress/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: nginx-ingress
-version: 1.15.1
+version: 1.15.2
```

Comment:
please bump the minor version to `1.16.0` as this is a new feature

Reply:
Ohh cool, just did it. Thanks